tests/test_activate.py

```diff
@@ -71,6 +71,10 @@
     'prompt',
 )
 
+ENV_VARS_FILE = '''ENV_ONE=one
```

Comment:
I think the changes to these tests make sense but I would really appreciate a second set of eyes on them in particular